using-mobile-extensions/adobe-journey-optimizer/api-reference.md

```diff
@@ -54,7 +54,7 @@ public static let extensionVersion
 
 ## handleNotificationResponse <a id="handlenotificationresponse"></a>
 
-Transmits the push notification interactions feedback.
+The handleNotificationResponse function transmits the push notification interactions' feedback.
```

Comment:
in my opinion, we can drop the `s` in `interactions` so it just becomes "...push notification interaction..."